software-engineer-in-test-US.md

```diff
@@ -0,0 +1,83 @@
+# Software Engineer in Test
+### Job Title
+ 
+Software Engineer in Test
+
+### Team
+Quality Engineering 
+
+### Location
+Remote (USA)
+
+### About Banked
+Banked intends to create the next global payments network. We believe credit and debit cards were created for a world that no longer exists, and technology will enable us to create a new, living payment network. We believe that the core purpose of a payment network is to enable sales and conversion, not to just move money and we're doing that with data in a way no one has seen before.<BR>
+COVID created a global pandemic, however, we are fortunate that COVID has meant growth in our business. As more transactions move online and as merchants focus more on improving margins our innovative, fundamentally different model for a payment network is gaining more and more traction.<BR>
+We've always enabled remote working as a company, and COVID has allowed us to embrace that even more; our team is now 100% distributed, and we believe it's the right way for us to continue to scale our mission. 
+
+### About the role and the team
+We're looking for someone who shares our vision, who identifies with our desire to reinvent payments and will help guide us through our growth to directly compete with Visa and Mastercard.<BR>
+You'll participate in the direction of the company and products, you'll be part of a small team where everyone has a voice.<BR>
+As a Software Engineer in Test at Banked, you'll be working on the integrity and quality of everything we build. From our customer-facing products to financial infrastructure moving money around the world.<BR>
+Someone who enjoys working as a team and wants to learn and grow will be most successful.<BR>
+We don't expect people to be experts with all of the technology we use or to immediately be able to pick up all the tasks we might ask of them. We want to build an environment where people can learn and grow openly, where failure is accepted as a part of learning and there's enough support to help people be successful.<BR>
+We are aiming to be a cross-functional team where Quality Engineers help developers to build quality products.
+
+### You will
+- Understand Banked products, their features and how users interact with them and develop scalable and reliable automated testing frameworks for testing Banked's software. This includes end-to-end web/api tests, service level tests, integration tests, contract tests, performance tests, infrastructure tests, chaos testing, security tests and exploratory tests.
+
+- Come up with Acceptance Test Plan during new market adoption and drive the test programme with third-party suppliers.
```

Comment:
Maybe reword this to something like
`Create acceptance test plans for testing new market entry, and drive the test programme with third-party suppliers.`

Could potentially re-word to mention alpha testing or something too like
`Create acceptance test plans for new market alpha testing, and drive the test programme with third-party suppliers.`